service/gcsutils/remotefs/types.go

```diff
@@ -22,7 +22,7 @@ type FileInfo struct {
 	NameVar    string
 	SizeVar    int64
 	ModeVar    os.FileMode
-	ModTimeVar time.Time
+	ModTimeVar int64 // Serialization of time.Time breaks in travis, so use an int
```

Comment:
oh, btw, it's not a compilation error. It's serialization/deserialization issue that causes my unit test to fail on travis but not my local machine